### PR TITLE
refactor: reduce the size of the docker container

### PR DESCRIPTION
**Pull request by enko**
I managed this by a multistaged build process and include only the stuff realy needed.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -46,7 +46,7 @@
     "semver": "^6.0.0",
     "sequelize": "^5.15.1",
     "sequelize-typescript": "^1.0.0",
-    "sharp": "^0.21.1",
+    "sharp": "^0.24.1",
```
Comment (turt2live) [inline package.json]: What does the `sharp` update help achieve?

File: Dockerfile [hunk 2]
```diff
@@ -1,47 +1,43 @@
...
 
-USER root
+WORKDIR /home/node/matrix-dimension
+
+COPY ./docker-entrypoint.sh /
 
-RUN apk del gcc make g++ && \
-    rm /home/node/matrix-dimension/Dockerfile && \
-    rm /home/node/matrix-dimension/docker-entrypoint.sh && \
-    dos2unix /docker-entrypoint.sh
```
Comment (turt2live) [inline Dockerfile]: please keep the dos2unix stuff, as it's used to test the image.

Comment (enko) [reply Dockerfile:35]: I have replaced dos2unix by a script to have minimal dependencies to the os of the container.

**Comment by enko** [comment]
@turt2live I addressed all the issues you have brought up, if time permits, you can have another look.

**Pull request merged**